src/components/connectors/Grid/shared.ts

```diff
@@ -1,5 +1,8 @@
 import type { TableIntlConfig } from 'src/types';
 
+export const CONNECTOR_GRID_COLUMNS = { xs: 4, sm: 4, md: 12, lg: 12, xl: 12 };
```

Comment:
Why are we setting `lg` and `xl`? Does the `md` setting not cascade?

Reply:
Closing the duplicate question and answering the first instance of it.